### DISCUSSION TITLE
set_installdir 用法

### DISCUSSION THREAD

**Discussion by DanielKui** [xmake-io/xmake#3987] [Q&A] [answered]
![image](https://github.com/xmake-io/xmake/assets/578118/cde103ea-4317-4069-84e9-a8dbe9a2864d)

请教一下，我在.lua里设置了  set_installdir("/home/kb/new/trunk/HXApi/bin/")
但使用xmake install的时候，为何还是复制到/usr/local目录呢？我是想复制到自定义目录"/home/kb/new/trunk/HXApi/bin/"
![image](https://github.com/xmake-io/xmake/assets/578118/be487179-2123-4366-920a-a67f2188b607)

**Comment by waruqi** [comment, marked as answer]
<img width="522" alt="image" src="https://github.com/xmake-io/xmake/assets/151335/6da89f07-8bd7-4014-b4e8-c5804d6d1f82">


我这里可以，没法复现
